src/ModelHelper.php

```diff
@@ -43,6 +43,17 @@ public function indexOnly(Model $model, $index_name)
         return !method_exists($model, 'indexOnly') || $model->indexOnly($index_name);
     }
 
+    public function wouldBeIndexed(Model $model, $index_name)
+    {
+        if (! method_exists($model, 'indexOnly')) {
+            return false;
+        }
+
+        $model->setRawAttributes($model->getOriginal());
```

Comment:
Yes, but it is not returned! It just modifies the Model locally to do some checks.